src/javascript/_cookies.js

```diff
@@ -0,0 +1,101 @@
+(function (window) {
+  var $ = window.$;
+  var Cookies = window.Cookies;
+  if (!Cookies || !$) {
+    return;
+  }
+
+  var COOKIE_NAMES = {
+    PREFERENCES_SET: 'cookies_preferences_set',
+    POLICY: 'cookies_policy'
+  };
+  
+  var DEFAULT_POLICY = {
+    essential: true,
+    settings: false,
+    usage: false
+  };
+
+  var GovUKCookie = {
+    get: function (name) {
+      var value = Cookies.get(name);
+      if (value) {
+        return JSON.parse(value);
+      }
+      return value;
+    },
+    set: function (name, value) {
+      var GOVUK_COOKIE_OPTIONS = {
+        expires: 365 // days
+      };
+    
+      return Cookies.set(
+        name,
+        value,
+        GOVUK_COOKIE_OPTIONS
+      );
+    }
+  };
+
+  GovUKCookie.set(
+    COOKIE_NAMES.POLICY,
+    DEFAULT_POLICY
+  );
+
+  var $cookieBanner = $('.global-cookie-message-dfe-sign-in');
```

Comment:
Would it not be better to use Ids here instead of classes?

Reply:
can be done as an extra safeguard actually; I'll add that in 😎 !